.github/workflows/ci.yml

```diff
@@ -34,27 +34,19 @@ concurrency:
 
 jobs:
   build:
-    name: Run Tests (Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }})
-    runs-on: ubuntu-latest
+    name: Run Tests (Julia ${{ matrix.version }} - ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
     strategy:
       # Don't cancel all running jobs when one job fails
       fail-fast: false
       matrix:
         version:
-          - '1.9'
-          - '1'
+          - 'min' # the earliest supported version compatible with the project
+          - '1'   # the latest stable 1.x release
```

Comment:
I don't see any reason to run tests for anything between the earliest supported version and latest release. This eliminates the need to change this file every time we increase the minimum version and therefore eliminates the risk of forgetting exactly this.